### PR TITLE
Add many new projects

### PR DESCRIPTION
**Pull request by p-wysocki**

### CODE REVIEW

**Comment by jiwaszki** [comment]
Can you also add https://github.com/zhuzilin/whisper-openvino ? Thanks!

**Comment by p-wysocki** [comment]
@jiwaszki done.

**Review by mlukasze — APPROVED** [review]

File: README.md
Comment (Pooja-B) [inline README.md:1]: @p-wysocki  Thanks for the PR. Could you please add a sentence about this project highlighting its functionality, similar to the other projects included.

**Review by Pooja-B — APPROVED** [review]
Thanks for adding these projects!

**Pull request merged**